com.unity.visualeffectgraph/Documentation~/Reference-Attributes.md

```diff
@@ -79,7 +79,7 @@ Setting a Lifetime attribute to a particle in an Initialize Context, will implic
 
 > Automatic particle reaping can be disabled by selecting the Update Context, then setting the Reap Particles value to **False**.
 
-> **Immortal Particles:** Particles with no lifetime will be considered as immortal. You can still kill them explicitly by setting their `alive` attribute to `false`.
+> **Immortal Particles:** Particles with no lifetime will are considered immortal. You can still cull the particles explicitly by setting their `alive` attribute to `false` but this doesn't kill them.
```

Comment:
This is incorrect. The line should be kept "You can still kill them explicitly by setting their `alive` attribute to `false`" with an additional one saying something like: "If the alive attribute is set to false in an output context, the particle will be culled from rendering but not killed"